### PR TITLE
Create Umami Recipe

### PR DESCRIPTION
**Pull request by kbarnhart**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by CJ-Wright** [comment]
This is one savory package.

**Comment by kbarnhart** [comment]
@conda-forge/staged-recipes this is ready for your review. 

I'm the only person listed as a maintainer and I confirm I'm willing to be there.

File: recipes/umami/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+
```
Comment (xhochy) [inline recipes/umami/meta.yaml]: Please remove the empty line

File: recipes/umami/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  sha256: 2d4a12da9f52df749511eb349d9033e9dac3596bb0792dbed2aeb49be53b917a
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python>=3.6
```
Comment (xhochy) [inline recipes/umami/meta.yaml]: Please add a space between package name and version spec (please do this for all requirements)
```suggestion
    - python >=3.6
```

File: recipes/umami/meta.yaml (hunk 1 above)
Comment (kbarnhart) [reply recipes/umami/meta.yaml]: Done

File: recipes/umami/meta.yaml (hunk 2 above)
Comment (kbarnhart) [reply recipes/umami/meta.yaml]: Done

**Comment by kbarnhart** [comment]
@xhochy thanks for taking the time to review this. 
I've made your requested changes. 

Happy to make any other changes recommended by others.

**Pull request merged**